tests/core/test_verifying.py

```diff
@@ -172,6 +172,7 @@ def test_ecr(seeder):
             registry=registry,
             sedge=ecredge,
             lei=LEI1,
+            role="EBA Data Submitter"
```

Comment:
Would prefer this to use an "EBA Data Submitter" constant here and in the other areas.